CHANGELOG/CHANGELOG-1.22.md

```diff
@@ -268,7 +268,7 @@
     - [CPU and Memory manager are not working correctly for Guaranteed Pods with multiple containers](#cpu-and-memory-manager-are-not-working-correctly-for-guaranteed-pods-with-multiple-containers)
     - [`CSIMigrationvSphere` feature gate has not migrated to new CRD APIs](#csimigrationvsphere-feature-gate-has-not-migrated-to-new-crd-apis)
     - [Workloads that saturate nodes with pods may see pods that fail due to node admission](#workloads-that-saturate-nodes-with-pods-may-see-pods-that-fail-due-to-node-admission)
-    - [Etcd v3.5.[0-2] data corruption](#etcd-v350-2-data-corruption)
+    - [Etcd v3.5.\[0-2\] data corruption](#etcd-v350-2-data-corruption)
```

Comment:
This is compatible with GitHub markdown preview and `\` is auto-added by my IDE.
However, this seems to be valid.